.github/workflows/golangci.yml

```diff
@@ -22,5 +22,5 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3.7.0
         with:
-          version: v1.53.3
+          version: v1.54.1
```

Comment:
required for the unparam in some way? (Or can be bumped via https://github.com/cosmos/ibc-go/pull/4394 as you'd done)

Reply:
can be bumped and isn't required for this.  Just required to commit.